projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/domain/linked-cases.model.ts

```diff
@@ -0,0 +1,72 @@
+export class LinkCaseReason {
+  key: string;
+  value_en: string;
+  value_cy: string;
+  hint_text_en: string;
+  hint_text_cy: string;
+  lov_order: number;
+  parent_key: string;
+  category_key: string;
+  parent_category: string;
+  active_flag: string;
+  child_nodes: string;
+  from: string;
+  selected?: boolean;
+}
+
+export class CCDCaseLinkType {
+  CaseReference: string;
+  CaseType: string;
+  CreatedDateTime: string
```

Comment:
@johnbenjamin-hmcts - missing semicolon

Reply:
@Daniel-Lam I have updated the code now.